charts/hlf-k8s/templates/job-bootstrap-orderer.yaml

```diff
@@ -62,11 +62,8 @@ spec:
             - { name: SECRET_NAME_CACERT, value: {{ .Values.secrets.caCert }} }
             - { name: SECRET_NAME_TLS_SERVER, value: {{ .Values.secrets.tls }} }
             - { name: SECRET_NAME_TLS_CLIENT, value: {{ .Values.secrets.tlsClient }} }
-            - { name: SECRET_NAME_TLS_SERVER_ROOT, value: {{ .Values.secrets.tlsRootCert }} }
-            - { name: SECRET_NAME_TLS_CLIENT_ROOT, value: {{ .Values.secrets.tlsClientRootCerts }} }
             - { name: SECRET_NAME_TLS_ORD_ROOT, value: {{ .Values.secrets.ordTlsRootCert }} }
             - { name: SECRET_NAME_ADMIN_CERT, value: {{ .Values.secrets.adminCert }} }
             - { name: SECRET_NAME_ADMIN_KEY, value: {{ .Values.secrets.adminKey }} }
-            - { name: SECRET_NAME_CA, value: {{ .Values.secrets.ca }} }
```

Comment:
Shouldn't we keep `SECRET_NAME_CA` and remove `SECRET_NAME_CACERT`?

Reply:
I kept used the "CA Cert" naming everywhere (as opposed to just "CA") because it seemed more explicit. Do you want me to use "CA" instead ?

Reply:
no it is ok for me